src/phantom.h

```diff
@@ -96,6 +97,8 @@ class Phantom : public QObject
 
     bool webdriverMode() const;
 
+    int remoteDebugPort() const;
```

Comment:
We already have this kind of property in `Config`. Why `Phantom` should has it too?